.github/workflows/build_container.yml

```diff
@@ -0,0 +1,44 @@
+name: publish
+
+on:
+  push:
```

Comment:
This should be ok to run only on `release`

Reply:
why?

Reply:
sometimes I need to deploy a hotfix from a PR or an unreleased fix. It's not practical to wait for the fix, besides GHCR is free for open source projects so it's simple and cheap to build everything all the time.

Reply:
Technically it is ok but it should be clear that we do not provide support for such images. And you can always build your hotfix yourself and ling your mirror.